### ISSUE TITLE
No sound in neo geo roms 

### ISSUE THREAD

**Issue by appletv4** [kevsmithpublic/MameAppleTV#2]
A lot of games have no sound , most of the neo geo games and King of fighters series 
You can hear the coin sound and the sound when you select player but no music from the game 
I am using the roms from 0.37b5

Also from that rom set Michael Jackson moonwalker game crashes the app

**Comment by appletv4** [comment]
Is this what I need for sound ?
http://dl.openhandhelds.org/cgi-bin/gp2x.cgi?0,0,0,0,5,2511

**Comment by kevsmithpublic** [comment]
No, The arm assembler code in the neogeo sound driver was commented out in order to get it working under arm64. This is a bug in the code and needs fixing.

Unfortunately I lack the skill at the moment to solve this.

If you look in ymdeltat.cpp line 240, this has been commented out and needs fixing.

**Comment by cheumadjeu** [comment]
sorry for the bad english , i have a french background .I don't know wether iam in the right place or not , I got a sound issue while playing kof98 .It seems i hit a key or multiples key ontop of my keyboard .,and it suddenly cut off.Could somebody help me .Please .I would like to turn it back on.Thanks .I am looking for something like a shortcut to fix the problem

**Comment by icpgithub** [comment]
Hey! Has this been fixed or someone knows how to fix it?
